src/theory/quantifiers/ce_guided_conjecture.h

```diff
@@ -0,0 +1,173 @@
+  std::vector< std::vector< Node > > d_ce_sk;
+  /** refinement lemmas */
+  std::vector< Node > d_refinement_lemmas;
+  std::vector< Node > d_refinement_lemmas_base;
+  /** quantified formula asserted */
+  Node d_quant;
+  /** quantified formula (after processing) */
+  Node d_embed_quant;
+  /** candidate information */
+  class CandidateInfo {
+  public:
+    CandidateInfo(){}
+    /** list of terms we have instantiated candidates with */
+    std::vector< Node > d_inst;
+  };
+  std::map< Node, CandidateInfo > d_cinfo;  
+  /** number of times we have called doRefine */
+  unsigned d_refine_count;
+  /** Whether we are syntax-guided (e.g. was the input in SyGuS format).
+   * This includes SyGuS inputs where no syntactic restrictions are provided.
+   */
+  bool d_syntax_guided;
+private: // for sygusStream
+  /** the streaming guards for sygus streaming mode */
+  std::vector< Node > d_stream_guards;
+  /** get current stream guard */
+  Node getCurrentStreamGuard() const;
+private: //non-syntax guided (deprecated)
+  /** the guard for non-syntax-guided synthesis */
+  Node d_nsg_guard;
+private:
+  /** convert node n based on deep embedding (Section 4 of Reynolds et al CAV 2015) */
+  Node convertToEmbedding( Node n, std::map< Node, Node >& synth_fun_vars, std::map< Node, Node >& visited );
+  /** collect constants */
+  void collectConstants( Node n, std::map< TypeNode, std::vector< Node > >& consts, std::map< Node, bool >& visited );
+  /** construct candidates */
+  bool constructCandidates( std::vector< Node >& clist, std::vector< Node >& model_values, 
+                            std::vector< Node >& candidate_values, std::vector< Node >& lems );
+  /** get candidadate */
+  Node getCandidate( unsigned int i ) { return d_candidates[i]; }
+  /** record instantiation (this is used to construct solutions later) */
+  void recordInstantiation( std::vector< Node >& vs ) {
+    Assert( vs.size()==d_candidates.size() );
+    for( unsigned i=0; i<vs.size(); i++ ){
+      d_cinfo[d_candidates[i]].d_inst.push_back( vs[i] );
+    }
+  }
+public:
+  CegConjecture( QuantifiersEngine * qe, context::Context* c );
+  ~CegConjecture();
+  /** get original version of conjecture */
+  Node getConjecture() { return d_quant; }
+  /** get deep embedding version of conjecture */
+  Node getEmbeddedConjecture() { return d_embed_quant; }
+  /** get next decision request */
+  Node getNextDecisionRequest( unsigned& priority );
+  /** increment the number of times we have successfully done candidate refinement */
+  void incrementRefineCount() { d_refine_count++; }
+  /** whether the conjecture is waiting for a call to do_Check below */
+  bool needsCheck( std::vector< Node >& lem );
+  /** whether the conjecture is waiting for a call to doRefine below */
+  bool needsRefinement();
+  /** get the list of candidates */
+  void getCandidateList( std::vector< Node >& clist, bool forceOrig = false );
+public:
```

Comment:
Same with the public blocks. public blocks should be on top the private section at the end.